tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/Test_WenoType.cpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "Evolution/DiscontinuousGalerkin/SlopeLimiters/WenoType.hpp"
+#include "Options/Options.hpp"
+#include "Options/ParseOptions.hpp"
+#include "Utilities/GetOutput.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace {
+struct ExampleLimiterType {
+  using type = SlopeLimiters::WenoType;
+  static constexpr OptionString help = {"Example help text"};
+};
+
+void check_weno_type_parse(const std::string& input,
```

Comment:
Could you try the `test_creation` function to see if it'll simplify this code a bit? It's in `tests/Unit/TestCreation.hpp`

Reply:
I am not getting having success here.

As far as I can tell, the only difference between my function `check_weno_type_parser` and `test_creation` is the additional `\n` character that `test_creation` inserts between the dummy class name and the input string. If I understand things correctly, this means that `test_creation` constructs classes that contain options, whereas my function constructs the option directly. Is my understanding correct?

Reply:
If my understanding is correct, one path forward would be to add a new `test_enum_creation` or similar that could be used here, in MinmodType (which has an essentially identical `check_minmod_type_parse`), and wherever else may be useful...?

Reply:
Okay, I'm fine with leaving as is for now or creating the `test_enum_creation` now and filing an issue :) Whichever you prefer